filebeat/harvester/reader/multiline.go

```diff
@@ -225,6 +227,8 @@ func (mlr *Multiline) clear() {
 // finalize writes the existing content into the returned message and resets all reader variables.
 func (mlr *Multiline) finalize() Message {
 
+	lines := common.MapStr{"lines": mlr.numLines}
+	mlr.message.AddFields(common.MapStr{"multiline": lines})
```

Comment:
Good point. Our current behaviour here is that is that fields will overwrite it. For json the behaviour does not change here. So in case someone has 
```
fields: 
  json: "Hello world"
```
already now, this would overwrite the fields from the json reader. The same applies `multiline`. So I think the behaviour didn't change even though I was never really thinking of the json overwriting by fields before. Seems like it wasn't an issue so far. The good thing is that we have it even mentioned in our docs: https://www.elastic.co/guide/en/beats/filebeat/1.3/configuration-filebeat-options.html#fields-under-root